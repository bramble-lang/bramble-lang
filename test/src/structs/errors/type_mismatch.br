fn my_main() -> i64 {
    let t: MyStruct := MyStruct{
        x: 5, 
        y: 1, 
        z: 10,
    };
    root::std::io::writei64ln(t.x);
    root::std::io::writei64ln(t.y);
    root::std::io::writeboolln(t.z);
    
    return;
}

struct MyStruct {
    x: i64,
    y: i64,
    z: bool,
}