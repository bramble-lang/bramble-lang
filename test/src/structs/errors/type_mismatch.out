Error: Semantic: L2: root::MyStruct.z expects bool but got i32
